.github/workflows/rust.yml

```diff
@@ -165,11 +189,39 @@ jobs:
         run: |
           git diff --staged --no-ext-diff --quiet --exit-code
 
+  publish-release:
+    name: Publish release
+    runs-on: ubuntu-latest
+    needs:
+      ["linux", "mac", "windows"]
+    if: ${{ !contains(needs.*.result, 'failure') && !contains(needs.*.result, 'cancelled') }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+        with:
+          merge-multiple: true
+      - id: release
+        if: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' }}
+        run: |
+          RELEASE_TAG=$(date "+%F-%H-%M")
```

Comment:
This could be problematic, if we have two updates in a day (it happens sometimes), so using commit hash might be better, but I don't know how we could bake this info in for https://github.com/servo/mozjs/pull/450. Can we get dependency commit hash using cargo without baking it in.

Reply:
Rusty_v8 does this using `CARGO_PKG_VERSION`:
https://github.com/denoland/rusty_v8/blob/9dd629e1c3c50fbbe70435ed50f6f09e9b69d7bb/build.rs#L320

Reply:
`CARGO_PKG_VERSION` seems like a better approach. I also hope this repo can provide version release again.
I'll update `release.yml` to use mozjs_sys's `CARGO_PKG_VERSION`, and it will skip the release if the version is already exist.

Reply:
I think `CARGO_PKG_VERSION` is actually the only way to go and because we moved all .c* files to mozjs-sys, the only thing that needs versions bumps is `mozjs-sys` when there are any C/C++ file changes.

It would be nice to make CI check for this too, but I think this can be done as a followup if (ever) needed. For now I think it would be enough if we test always latest release on push to main (instead of testing release only if it's new).